### DISCUSSION TITLE
大佬，打包的时候，不想把安装包一起打包，怎么操作啊？

### DISCUSSION THREAD

**Discussion by hackhase** [lisonge/vite-plugin-monkey#129] [General]
'element-plus': cdn.jsdelivr('ElementPlus', 'dist/index.full.min.js')，按这个操作后，老是出现 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'defineComponent')这个错误

**Comment by lisonge** [comment]
https://github.com/lisonge/vite-plugin-monkey/blob/main/README_zh.md#%E9%80%9A%E8%BF%87-require-%E5%8A%A0%E8%BD%BD%E7%9A%84-iife-%E5%92%8C-umd-%E6%B7%B7%E7%94%A8%E7%9A%84%E9%97%AE%E9%A2%98
